src/Discord.Net.WebSocket/Entities/Channels/SocketVoiceChannel.cs

```diff
@@ -52,7 +55,10 @@ public override SocketGuildUser GetUser(ulong id)
                 return user;
             return null;
         }
-        
+
+        public Task<ICategoryChannel> GetCategoryAsync(RequestOptions options = null)
```

Comment:
Remove this and leave only INestedChannel.GetCategoryAsync() since theres already the Category property to retrieve it.

Same pattern as how we deal to retrieve users.